### ISSUE TITLE
Modifier key combination output

### ISSUE THREAD

**Issue by coderyangyangyang** [donniebreve/touchcursor-linux#10]
Really appreciate your code，it works well. You know in the Windows touchcursor, you can convert something like Space+x to Ctrl+y. Could you add this key combination remaping function to your code？

**Comment by donniebreve** [comment]
The plan is to eventually have all the keys configurable from a conf file. I think your specific request would be to emit a modifier key and then a regular key for some keys, which I think would not be too hard. I will keep it in the back of my head while implementing the conf file.

**Comment by AlienKevin** [comment]
I second that. It would be nice if I can remap space+h from page up to enter.

**Comment by donniebreve** [comment]
You should be able to remap space+h->enter now. The modifier key that coderyangyang asked for still needs some work.

**Comment by RomanRcT** [comment]
Hello, how are you doing with this feature? It is needed to remap VIM-like keybinding everywhere, like Y - copy, P - paste.

**Comment by Adda0** [comment]
Hey, as far as I know, it is still not possible to emit both a modifier key and an additional normal key at once by a single key press. Or am I wrong, @donniebreve? I have some modifier keys mapped as follows:

- `D` to `Ctrl` and
- `S` to `Alt`.

If curious, see [my dotfiles repo](https://github.com/Adda0/dotfiles/blob/31297f3a0734e764aca663b3c3577b1225b44d59/home/.config/touchcursor/touchcursor.conf#L29) for the configuration. It makes a huge difference for me by itself, but I must agree the requested functionality would be useful, too.

**Comment by Adda0** [comment]
Another update.

I have implemented the feature requested by @coderyangyangyang in my C++ port (work-in-progress, nevertheless a working prototype). As of now, one can do mappings such as the following:
```haskell
KEY_S=KEY_LEFTSHIFT+KEY_G
KEY_W=KEY_T,KEY_G
KEY_H=KEY_LEFTSHIFT+KEY_G
KEY_J=KEY_T,KEY_F
KEY_I=KEY_T,KEY_LEFTSHIFT+KEY_G,KEY_F,KEY_LEFTCTRL+KEY_V,KEY_F
```
`<key>,<key>` emits a sequence of keys (in the given order). For example, from the previous list, when pressing hyper key and key `W`, `KEY_T,KEY_G` would emit `tg`. `<key>+<key>` emits a combination of keys simultaneously pressed. That is, both keys are simultaneously pressed, then all released at the same time. For example, pressing hyper key with key `H` emits `G` (as in `shift` + `g`). Both sequences and combinations can be combined, of course.

Do you all think this functionality meets the requirements described in this issue? And of course, feel free to experiment with the [WIP prototype](https://github.com/Adda0/touchcursor-linux/tree/cpp-port), installed following the [instructions](https://github.com/Adda0/touchcursor-linux/tree/cpp-port#how-to-install=), if you want to.

**Comment by magnus-ISU** [comment]
Also implemented in #59 

It seems like Adda0 has a more fully-featured implementation. However, I spent some time earlier and couldn't get theirs to work on my computer. Plus #59 has a decent shot at getting merged.

**Comment by Adda0** [comment]
> It seems like Adda0 has a more fully-featured implementation. However, I spent some time earlier and couldn't get theirs to work on my computer.

I would be interested in seeing what did not work out for you. I am using the linked version on my computers, so I would like to know what happened in your case, if you feel like analysing what went wrong on your side. If so, feel free to open an issue on my fork to discuss this further.

**Comment by auouymous** [comment]
This was implemented in #59, right?

**Comment by donniebreve** [comment]
Yes I believe it was.
